.azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+pr: ["master"]
+
+jobs:
+  - job:
+    strategy:
+      matrix:
+        mac:
+          imageName: 'macos-10.13'
+        windows:
+          imageName: 'vs2017-win2016'
+    pool:
```

Comment:
Any special reason there's no Linux build?

Reply:
because I copied this from druid and we don't build druid on linux. But yea, should add that I suppose.